### PR TITLE
Configure Jest for TypeScript tests

### PR DESCRIPTION
**Pull request by DesignerDave**
## ✍️ Proposed changes

Previously, tests written in `ts` or `tsx` wouldn't run. This fixes that issue.

## 🛠 Types of changes

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## ✅ Checklist

- [ ] ~~I have added tests that prove my fix is effective or that my feature works~~ N/A
- [ ] ~~I have added necessary documentation (if appropriate)~~ N/A

### CODE REVIEW

File: jest.config.js [hunk 1]
```diff
@@ -107,7 +118,7 @@ module.exports = {
   // restoreMocks: false,
 
   // The root directory that Jest should scan for tests and modules within
-  // rootDir: null,
+  // rootDir: "./packages/",
```
Comment (hswolff) [inline jest.config.js]: o_O this is commented out.

Comment (DesignerDave) [reply jest.config.js]: Supposed to be – I was playing with the option, but we don't need it.

Comment (DesignerDave) [reply jest.config.js]: That said, I can certainly put the comment back at the default, or remove all comments. I didn't mean to change this line.

**Comment by DesignerDave** [comment]
@hswolff CI was giving me the following error:
```
npm WARN prepare removing existing node_modules/ before installation
npm ERR! cipm can only install packages when your package.json and package-lock.json or npm-shrinkwrap.json are in sync. Please update your lock file with `npm install` before continuing.
```

I thought that the issue might've been something I can resolve so I did run `npm i`, but now I think that it may be because of lerna-hoisted packages. Thoughts? It was failing prior to the `package-lock.json` change as well.

File: jest.config.js (hunk 1 above)
Comment (DesignerDave) [reply jest.config.js]: Update: it's now back to the default value in the comment. There's an argument that we should consider removing the unused options in this file, but I find it's nice to have inline documentation on the defaults for each. I could, however, be easily swayed.

Comment (hswolff) [reply jest.config.js]: Yeah I'm fine either way. No real preference. Could remove and always look up the options later.

**Comment by hswolff** [comment]
I had this happen too and managed to fix it by re-generating package-lock. Maybe let's add emotion and create-emotion to the root repo? Or can try `rm -rf node_modules package-lock.json && npm install` which seemed to fix it last time for me.

**Comment by DesignerDave** [comment]
I'm gonna add the dependencies to the root. I don't think it's a great flow if we have to remove `package-lock` and install before every push.

File: package.json [hunk 2]
```diff
@@ -96,6 +96,9 @@
     "typescript": "^3.4.3",
     "url-loader": "1.1.2",
     "webpack": "4.26.0",
-    "webpack-cli": "3.1.2"
+    "webpack-cli": "3.1.2",
+    "create-emotion": "^10.0.7",
```
Comment (hswolff) [inline package.json]: should be alphabetical. if you do it via npm cli it does it for you.

**Pull request merged**